tests/acceptance/search-test.js

```diff
@@ -0,0 +1,42 @@
+import { test } from 'qunit';
+import moduleForAcceptance from 'job-search-app/tests/helpers/module-for-acceptance';
+import Ember from 'ember';
+
+let GatekeeperStub = Ember.Service.extend({
+  on() { },
+  off() { },
+  isSignedIn: true
+});
+
+moduleForAcceptance('Acceptance | login',  {
+  beforeEach() {
+    this.application.register('service:gatekeeperStub', GatekeeperStub);
+    this.application.inject('route:search', 'gatekeeper', 'service:gatekeeperStub');
+    this.application.inject('route:detailed-search', 'gatekeeper', 'service:gatekeeperStub');
+  }
+});
+
+test('A user can view profile based jobs', function(assert) {
+  visit('/search');
+
+  andThen(function() {
+    assert.equal(currentURL(), '/search');
+    // assert.equal(Ember.$(".job-entry:visible").length, 0)
+    click("#recommended-jobs-open-close");
+    // andThen() {
+    //     assert.equal(Ember.$(".job-entry:visible").length, 2)
```

Comment:
Does this check that the job list is visible and has a length of 2? Sorry I only briefly did something similar in the ember tutorial.

Reply:
Yes, it will do that once I finish the test :)